slow-restart/README.md

```diff
@@ -0,0 +1,29 @@
+# slow-restart
```

Comment:
I would rather name it smth like `linkerd-dataplane-updater`

Reply:
I'll go with `linkerd-dataplane-updater`